src/Mpdf.php

```diff
@@ -10448,7 +10449,7 @@ function _dounderline($x, $y, $txt, $OTLdata = false, $textvar = 0)
 	/* -- WATERMARK -- */
 
 	// add a watermark
-	function watermark($texte, $angle = 45, $fontsize = 96, $alpha = 0.2)
```

Comment:
As internal methods miss visibility definition, changing `watermark` method signature by removing a parameter is a no-go or this could be merged at soonest in the next major release as a breaking change. Keep the method signature as it was.